app.go

```diff
@@ -438,3 +449,81 @@ func executeCancelSell(cmd command) bool {
 
 	return true
 }
+
+func executeSetBuyAmount(cmd command) bool{
+	account := accountStore.GetAccount(cmd.UserID)
+
+	if account == nil {
+		log.Infof("User %s does not have an account", cmd.UserID)
+		return false
+	}
+
+	amount, err := currency.NewFromString(cmd.Args[1])
+	if err != nil {
+		// Bail on parse failure
+		log.Error("Failed to parse currency")
```

Comment:
Not a super great log message. I know I probably made the same one a while ago, but it doesn't refer to a user or instance or anything. We can leave it for now, but later it should be refined.